### DISCUSSION TITLE
Long press to save/copy image

### DISCUSSION THREAD

**Discussion by Mitch557** [aeharding/voyager#177] [Ideas]
Had a thought as I was browsing and thought I'd share. Currently on iOS when you long press in an image it hovers, allowing you to drag the image around. It would be nice if this was changed to open a menu with options to share, save to photos, copy, etc. Ideally this would share/copy the image itself and not a link to the post. 

Current functionality:
![image](https://github.com/aeharding/wefwef/assets/42188381/cdd7c6e8-03cd-494d-a081-6d54716cc502)

Proposed functionality:
![image](https://github.com/aeharding/wefwef/assets/42188381/bd0e9ab9-8aef-4f6c-9edc-b78772c12852)

**Comment by adjoseph16** [comment]
Just in case the messaging got lost here among the many other updates, I just wanted to let you know @Mitch557 that this is now implemented! Not by me, I just noticed it and remembered this PR :)

**Reply by Mitch557** [reply]
Just checked it out and it works like a charm. Thank you!
